tests/integration/test_data/global_update_channel_1/recipe.yaml

```diff
@@ -0,0 +1,39 @@
+recipe:
```

Comment:
Can you add two recipes with two different versions. And adapt the nushell script accordingly so that it runs the update correctly

Reply:
When `rattler-build` allows variants on the version this would be an instant fix.